src/test/java/com/jcabi/dynamo/mock/H2DataTest.java

```diff
@@ -134,7 +134,25 @@ public void createsTablesWithLongNames() throws Exception {
      */
     @Test
     public void supportsTableNamesWithIllegalCharacters() throws Exception {
-        new H2Data().with(".-.", new String[] {"pk"}, new String[0]);
+        new H2Data().with(".-.", new String[]{"pk"}, new String[0]);
+    }
+
+    /**
+     * H2Data supports column names with characters illegal to H2.
+     * todo: #28:30min H2Data doesn't support COLUMNS with ".", "-" or digits
+     * but I don't know for sure should it support these symbols or not.
+     * It's needed to be confirmed and test should be uncommented
+     * when H2Data will be supporting mentioned symbols.
+     * @throws Exception In case test fails
+     */
+    @Test
+    @Ignore
+    public void supportsColumnNamesWithIllegalCharacters() throws Exception {
+        final String key = "0-.col.-0";
```

Comment:
I don't see where does this value exist in the file, have you inlined this variable and removed the `final String key`?
Same for the test table below, I don't see `"test"` string anywhere in the file, so if you inline it it should work (of course you have to remove the `table` variable)